k8s/k8s.go

```diff
@@ -1321,3 +1353,40 @@ func loadClientFor(config *rest.Config) (*kubernetes.Clientset, *rest.RESTClient
 func roundUpSize(volumeSizeBytes int64, allocationUnitBytes int64) int64 {
 	return (volumeSizeBytes + allocationUnitBytes - 1) / allocationUnitBytes
 }
+
+// GetLocalIPList returns the list of local IP addresses, and optionally includes local hostname.
+func GetLocalIPList(includeHostname bool) ([]string, error) {
+	ifaces, err := net.Interfaces()
+	if err != nil {
+		return nil, err
+	}
+	ipList := make([]string, 0, len(ifaces))
+	for _, i := range ifaces {
+		addrs, err := i.Addrs()
+		if err != nil {
+			return ipList, fmt.Errorf("Error listing addresses for %s: %s", i.Name, err)
```

Comment:
We should either return nil in the ipList here or If we want to send IP addresses of possible interfaces then we should have a continue instead of return here.